tests/codegen/map_declarations.cpp

```diff
@@ -0,0 +1,29 @@
+#include "common.h"
+
+namespace bpftrace::test::codegen {
+
+TEST(codegen, map_declarations)
```

Comment:
Maybe break into separate tests to make catching strange issues a bit easier? E.g. if *one* of the map types change, then something is probably up.